modules/jaeger-config-query.adoc

```diff
@@ -0,0 +1,57 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-query_{context}"]
+= Jaeger Query configuration options
+
+Query is a service that retrieves traces from storage and hosts the user interface to display them.
+
+.Jaeger Query parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+|spec:
+  query:
+   options: {}
+   resources: {}
+|Configuration options that define the Query service.
+|
+|
+
+|query:additional-headers
+|Additional HTTP response headers. Can be specified multiple times.
+|Format: "Key: Value"
+|
+
+|query:base-path:
+|The base path for all jaeger-query HTTP routes can be set to a non-root value, for example, `/jaeger` would cause all UI URLs to start with `/jaeger`. This can be useful when running jaeger-query behind a reverse proxy.
+|/{path}
+|
+
+|query:port
+|The port for the query service
+|
+|16686
+|===
+
+.Sample Query configuration
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "my-jaeger"
+spec:
+  strategy: allInOne
+  allInOne:
+    image: jaegertracing/all-in-one:{ProductVersion}
```

Comment:
We need to avoid including the `image:` property in the examples or docs. Only the productised images are supported.